src/main/java/com/iota/iri/network/spam/DropFixedPercentage.java

```diff
@@ -0,0 +1,81 @@
+package com.iota.iri.network.spam;
+
+import com.iota.iri.network.Neighbor;
+import com.iota.iri.utils.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Simple spam prevention strategy that considers a percentage of the most active neighbors (the ones that send the
+ * most new transactions) to be spamming.
+ */
+public class DropFixedPercentage implements SpamPreventionStrategy {
+
+    // needs to be static according to IRI code style
+    private static final Logger LOG = LoggerFactory.getLogger(DropFixedPercentage.class);
+
+    private final int dropPercentage;
+    private Map<Neighbor, Long> transactionCount;
+    private Set<Neighbor> mostActiveNeighbors = Collections.emptySet();
+
+    /**
+     * Creates a new spam strategy that marks a fixed percentage of neighbors to be spamming.
+     * @param dropPercentage The percentage of neighbors that are considered to be spamming. Percentage will be rounded
+     *                       down. So 33% of 3 neighbors are 0 neighbors. 34% of 3 neighbors is one neighbor.
+     * @param neighbors The list of neighbors that should be examined for finding spamming nodes.
+     */
+    public DropFixedPercentage(int dropPercentage, List<Neighbor> neighbors) {
+        LOG.info("Created fixed percentage spam prevention strategy. Dropping {}% of most active neighbors.", dropPercentage);
+        this.dropPercentage = dropPercentage;
+        transactionCount = toMap(neighbors);
+    }
+
+    /**
+     * Recalculates what neighbors are currently spamming. The most active neighbors since the last call of this
+     * method will be marked to be spamming.
+     * @param neighbors The list of neighbors that should be examined for spamming neighbors.
+     */
+    public void calculateSpam(List<Neighbor> neighbors) {
+        List<Pair<Long, Neighbor>> deltas = new ArrayList<>(neighbors.size());
+        neighbors.forEach(neighbor -> {
+            Long previous = transactionCount.getOrDefault(neighbor, neighbor.getNumberOfNewTransactions());
+            long delta = neighbor.getNumberOfNewTransactions() - previous;
+            if (delta > 0) { // only active neighbors count
+                deltas.add(new Pair<>(delta, neighbor));
+            }
+        });
+        Collections.shuffle(deltas); // in case we have similar deltas
```

Comment:
In case two neighbors have the same count of incoming messages without shuffling always the same neighbor would get selected. With shuffling a random neighbor will get selected.